dist/unix/systemd/qbittorrent-nox.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=qBittorrent-nox service
+Documentation=man:qbittorrent-nox(1)
+Wants=network-online.target
```

Comment:
`network-online.target` doesn't work for user services unfortunately. There are some workaround though.

https://github.com/systemd/systemd/issues/28762